src/ui/zcl_abapgit_gui_page_repo_over.clas.abap

```diff
@@ -539,70 +555,70 @@ CLASS zcl_abapgit_gui_page_repo_over IMPLEMENTATION.
     CLEAR mt_col_spec.
 
     _add_column(
-      iv_tech_name = 'FAVORITE'
-      iv_css_class = 'wmin'
+      iv_tech_name      = 'FAVORITE'
+      iv_css_class      = 'wmin'
       iv_allow_order_by = abap_false ).
 
     _add_column(
-      iv_tech_name = 'TYPE'
-      iv_css_class = 'wmin'
+      iv_tech_name      = 'TYPE'
+      iv_css_class      = 'wmin'
       iv_allow_order_by = abap_false ).
 
     _add_column(
-      iv_tech_name = 'NAME'
-      iv_display_name = 'Name'
+      iv_tech_name      = 'NAME'
+      iv_display_name   = 'Name'
       iv_allow_order_by = abap_true ).
 
     _add_column(
-      iv_tech_name = 'PACKAGE'
-      iv_display_name = 'Package'
-      iv_css_class = 'package'
+      iv_tech_name      = 'PACKAGE'
+      iv_display_name   = 'Package'
+      iv_css_class      = 'package'
       iv_allow_order_by = abap_true ).
 
     _add_column(
-      iv_tech_name = 'URL'
-      iv_display_name = 'Remote'
+      iv_tech_name      = 'URL'
+      iv_display_name   = 'Remote'
       iv_allow_order_by = abap_true ).
 
     _add_column(
-      iv_tech_name = 'BRANCH'
-      iv_display_name = 'Branch'
+      iv_tech_name      = 'BRANCH'
+      iv_display_name   = 'Branch'
       iv_allow_order_by = abap_true ).
 
     _add_column(
-      iv_tech_name = 'DESERIALIZED_BY'
-      iv_display_name = 'Deserialized by'
-      iv_css_class = 'ro-detail'
+      iv_tech_name      = 'DESERIALIZED_BY'
+      iv_display_name   = 'Deserialized by'
+      iv_css_class      = 'ro-detail'
       iv_allow_order_by = abap_true ).
 
     _add_column(
-      iv_tech_name = 'DESERIALIZED_AT'
-      iv_display_name = 'Deserialized at'
-      iv_css_class = 'ro-detail'
+      iv_tech_name      = 'DESERIALIZED_AT'
+      iv_display_name   = 'Deserialized at'
+      iv_css_class      = 'ro-detail'
       iv_allow_order_by = abap_true ).
 
     _add_column(
-      iv_tech_name = 'CREATED_BY'
-      iv_display_name = 'Created by'
-      iv_css_class = 'ro-detail'
+      iv_tech_name      = 'CREATED_BY'
+      iv_display_name   = 'Created by'
+      iv_css_class      = 'ro-detail'
       iv_allow_order_by = abap_true ).
 
     _add_column(
-      iv_tech_name = 'CREATED_AT'
-      iv_display_name = 'Created at'
-      iv_css_class = 'ro-detail'
-      iv_add_tz = abap_true
+      iv_tech_name      = 'CREATED_AT'
+      iv_display_name   = 'Created at'
+      iv_css_class      = 'ro-detail'
+      iv_add_tz         = abap_true
       iv_allow_order_by = abap_true ).
 
     _add_column(
-      iv_tech_name = 'KEY'
-      iv_display_name = 'Key'
-      iv_css_class = 'ro-detail'
+      iv_tech_name      = 'KEY'
+      iv_display_name   = 'Key'
+      iv_css_class      = 'ro-detail'
       iv_allow_order_by = abap_true ).
 
     _add_column(
-      iv_tech_name = 'GO'
-      iv_css_class = 'ro-go'
+      iv_tech_name      = 'GO'
+      iv_css_class      = 'ro-go'
```

Comment:
This is caused by a new option
![image](https://user-images.githubusercontent.com/5097067/135684855-e889b4f0-39fe-47a9-a8a2-4d5026010055.png)

I can undo the formatting if it's undesired, but as long as it is done automatically i think this is better